src/guide/installation.md

```diff
@@ -30,6 +30,8 @@ Simply download and include with a script tag. `Vue` will be registered as a glo
 
 Available on [jsdelivr](//cdn.jsdelivr.net/vue/{{vue_version}}/vue.min.js) or [cdnjs](//cdnjs.cloudflare.com/ajax/libs/vue/{{vue_version}}/vue.min.js) (takes some time to sync so the latest version might not be available yet).
 
+Also available on [npmcdn](https://npmcdn.com/vue/vue.min.js), which will reflect the latest version as soon as it is published to npm.
```

Comment:
Thanks, small correction: the link should be `https://npmcdn.com/vue/dist/vue.min.js`